README.md

```diff
@@ -0,0 +1,21 @@
+# census
+
+Serverless deployment steps:
+<ul>  
+
+<li>Adjust implementation logic in handler.js
+<li>Adjust serverless.yml accordingly with function name
+
+</ul>
+Type the following into the command line:
+<ol>
+<li>serverless config credentials --provider aws --key *YOUR KEY* --secret *YOUR SECRET KEY* --overwrite</bold>
+<li>serverless deploy
+
+</ol>
+
+
+
```

Comment:
* is there anything else that someone using this project would need to do after pulling the repo to get their environment set up? like maybe they need to make sure to have run `npm install -g serverless`. can we include those types of instructions?
* it would be helpful to also have instructions to run it locally before deploying: 
this tutorial that helps with the rds setup also gives some commands to run for local testing with the serverless-offline plugin under the "testing" section https://medium.com/swlh/creating-a-serverless-rest-api-with-node-js-aws-lambda-api-gateway-rds-and-postgresql-303b0baac834#testing